Assets/MRTK/StandardAssets/EditorUtilities/OnLoadUtilities.cs

```diff
@@ -0,0 +1,110 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.IO;
+using UnityEditor;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Utilities.Editor
+{
+    [InitializeOnLoad]
+    static class OnLoadUtilities
+    {
+        private const string SessionStateKey = "StandardAssetsOnLoadUtilitiesSessionStateKey";
+
+        private const string ShaderSentinelGuid = "05852dd420bb9ec4cb7318bfa529d37c";
+        private const string ShaderSentinelFile = "MRTK.Shaders.Sentinel";
```

Comment:
I'm not sure if I should use a closed MR for this, but I did run into a problem with this script when running Unity on MacOS.  On MacOS `DirectoryInfo.GetFiles` appears to be case-sensitive even if the underlying filesystem is not. The sentinel file is actually named `MRTK.Shaders.sentinel` with a lowercase `s` on `Sentinel`. On MacOS the sentinel file wasn't found, so the exception about not being able to find the source for the shaders is thrown.

The simple fix is changing this constant to use the lowercase `s` for `sentinel`.

Reply:
This will be fixed in the upcoming 2.5.2 release, via https://github.com/microsoft/MixedRealityToolkit-Unity/pull/9005